src/lib/common/routing/routing.ts

```diff
@@ -74,7 +74,7 @@ export function SetRoutingUnits(routing: Routing, token: BridgeToken | undefined
     if (!token) throw new Error("Token not defined");
     if (routing.units) return;
     if (!routing.amount) throw new Error("Routing amount not defined");
-    if (!token.decimals) throw new Error("Routing decimals not defined");
+    if (token.decimals == undefined) throw new Error("Routing decimals not defined");
```

Comment:
@aurel-fr should we add a `0` or negative validation as well?

Reply:
You could definitely add checks to make sure the decimal is a uint, esp if this is the component that talks to validators. `0` is fine though